tao_compiler/mlir/disc/transforms/conv_rewriter.cc

```diff
@@ -444,12 +510,238 @@ struct DiscConvRewriterPass
   int cc_minor_;
 };
 
+// The basic logic of this pass is to insert transpose op to ensure the conv op
+// having expected format.
+struct DiscQuantizedConvRewriterPass
```

Comment:
What's the difference between this pass and the normal pass?

Reply:
ditto :)